integration-cli/docker_cli_network_unix_test.go

```diff
@@ -699,8 +698,6 @@ func (s *DockerNetworkSuite) TestDockerNetworkInspectDefault(c *check.C) {
 	c.Assert(nr.EnableIPv6, checker.Equals, false)
 	c.Assert(nr.IPAM.Driver, checker.Equals, "default")
 	c.Assert(len(nr.IPAM.Config), checker.Equals, 1)
-	c.Assert(nr.IPAM.Config[0].Subnet, checker.NotNil)
```

Comment:
Perhaps we should create a tracking issue; I can imagine this was meant to check for a non-empty value, so could be changed to, e.g. `nr.IPAM.Config[0].Subnet != ""`